web/src/typing_events.ts

```diff
@@ -91,14 +120,17 @@ function get_key(event) {
             key = typing_data.get_direct_message_conversation_key(recipients);
             break;
         default:
-            blueslip.error("Unexpected message_type in typing event: " + event.message_type);
+            blueslip.error("Unexpected message_type in typing event.");
```

Comment:
TS can not access properties of type `never`( `event` object here)

Reply:
But `event` is a `TypingEvent`, never `undefined.` This code should be fine.

Reply:
Also, this `default` case can be deleted; the types will enforce that the value must be `stream` or `direct`.

Reply:
Yes , I agree , the `switch` case code block should be replaced with `if-else `statements . I will add a prep commit doing that, I guess